lib/play/game_page.dart

```diff
@@ -650,6 +655,11 @@ class _GamePageState extends State<GamePage> {
     _log.fine(
         '[${_state.name}] got counting result: winner=${res.winner} lead=${res.scoreLead}');
 
+    if (_state == GameState.over) {
```

Comment:
Not strictly necessary - I updated OGSGame to not fire counting results out of the "stone removal" phase.  But I figured, it still doesn't make sense to leave the `GameState.over` state if `result` is set.